crates/orderbook/openapi.yml

```diff
@@ -25,6 +25,9 @@ paths:
   /api/v1/orders:
     post:
       summary: Create a new order.
+        If the appData contains a valid replacement order UID, then the indicated order is cancelled. 
+        This allows an old order to be cancelled AND a new order to be created in an atomic operation with a single signature.
+        This may be useful for replacing orders when on-chain prices move outside of the original order's limit price.
```

Comment:
Can this be somehow made clearer as to what and how needs to be provided?
It mentions the replacement order UID in the appData, but doesn't say which fields or how it should be presented.

Unless one knows what this is about, there's little guidance as to how actually perform the order replacement.